main.tf

```diff
@@ -1 +1,306 @@
-#(empty)
+resource "aws_wafv2_web_acl" "main" {
+  name        = var.name
+  description = var.description
+  scope       = var.scope
+  tags        = var.tags
+
+  dynamic "default_action" {
```

Comment:
Hi Patrick, I see we have two `default_action` dynamic block?

Reply:
Hey @ndegwajohn  the first block is for creating an empty `allow {}` block when a default_action is not defined by user since this is required

Reply:
Gotcha.
Since the the web UI I see only one block for for `Default web ACL action for requests that don't match any rules`, I think the dynamic block should start from allow, not default action, what do you think?